start-site/src/main/java/io/spring/start/site/extension/dependency/testcontainers/TestcontainersProjectGenerationConfiguration.java

```diff
@@ -68,6 +69,20 @@ public ImplicitDependencyHelpDocumentCustomizer testcontainersHelpCustomizer(Bui
 
 	@Configuration(proxyBeanMethods = false)
 	@ConditionalOnPlatformVersion("3.1.0-RC1")
+	static class ServiceConnectionsConfiguration {
```

Comment:
I think this should move to a position below where it's imported or perhaps things should be restructured a bit (see my other comment).